src/wp-includes/kses.php

```diff
@@ -1468,6 +1470,41 @@ function wp_kses_hair( $attr, $allowed_protocols ) {
 	return $attrarr;
 }
 
+/**
+ * Santizes uris in attributes
+ *
+ * This condenses code that was spread around two functions and several cases. It places the list of attributes
+ * which can have uris in them and checks for ones that can have multiple uri candidates (like srcset)
+ * It ultimately passes everything to wp_kses_bad_protocol() to do the actual work.
+ *
+ * @since ?
+ *
+ * @param string $attrname Attribute name to test against.
+ * @param string $attrvalue Content to filter bad protocols from.
+ * @param string[] $allowed_protocols Array of allowed URL protocols.
+ * @return string Filtered content.
+ */
+function wp_kses_sanitize_uris( $attrname, $attrvalue, $allowed_protocols ) {
+	$uris           = wp_kses_uri_attributes();
+	$uri_candidates = array( 'srcset' );
+
+	if ( ! in_array( strtolower( $attrname ), $uris ) ) {
+		return $attrvalue;
+	} else {
+		if ( in_array( strtolower( $attrname ), $uri_candidates ) ) {
+			$thesevals = preg_split( '/\s*,\s+/', $attrvalue );
```

Comment:
Seems the regex here should be `/\s*,\s*/` as `\s+` might miss some cases where URIs are split only with a comma and no space?
```suggestion
			$thesevals = preg_split( '/\s*,\s*/', $attrvalue );
```